packages/tenant-process/test/tenant.integration.test.ts

```diff
@@ -115,9 +137,268 @@ describe("Integration tests", () => {
   });
 
   describe("tenantService", () => {
-    describe("tenant creation", () => {
-      it("TO DO", () => {
-        expect(1).toBe(1);
+    describe("createTenant", () => {
+      it("Should create a tenant", async () => {
+        const kind = tenantKind.PA;
+        const selfcareId = generateId();
+        const tenantSeed: ApiSelfcareTenantSeed = {
+          externalId: {
+            origin: "IPA",
+            value: "123456",
+          },
+          name: "A tenant",
+          selfcareId,
+        };
+        const id = await tenantService.createTenant(tenantSeed, [], kind);
```

Comment:
You shouldn't test this, the function to be tested is `selfcareUpsertTenant` which is called by the router.

Moreover `createTenant` seems to be unused, in that case we can safely remove it